src/push.js

```diff
@@ -68,14 +68,15 @@ function getAuthToken(endpoint) {
  */
 function addAuthToken(pattern, token) {
   authTokens.push({pattern, token});
-};
+}
 
 /**
  * Sends a message using the Web Push protocol
  * @param  {String}   message      The message to send
  * @param  {Object}   subscription The subscription details for the client we
  *                                 are sending to
- * @return {Promise}
+ * @return {Promise} A promise that resolves if the push was sent successfully
```

Comment:
This is one of those places where it is actually really useful to have the return value specified. :+1: